docs/dev/operators.md

```diff
@@ -0,0 +1,103 @@
+# Second Level Operator integration with CVO
+
+## How do I get added to the release payload?
+Add the following to your Dockerfile
+```
+FROM …
+
+ADD manifests-for-operator/ /manifests
+LABEL io.openshift.release.operator=true
+```
+
+Ensure your image is published into the cluster release tag by ci-operator
+Wait for a new release payload to be created (usually once you push to master in your operator).
+
+## What do I put in /manifests?
+
+You need the following:
+
+1..N manifest yaml or JSON files (preferably YAML for readability) that deploy your operator, including:
+
+  - Namespace for your operator
+  - Roles your operator needs
+  - A service account and a service account role binding
+  - Deployment for your operator
+  - An OperatorStatus CR
+  - Any other config objects your operator might need
+  - An image-references file (See below)
+
+In your deployment you can reference the latest development version of your operator image (quay.io/openshift/origin-machine-api-operator:latest).  If you have other hard-coded image strings, try to put them as environment variables on your deployment or as a config map.
+
+Your manifests will be applied in alphabetical order by the CVO, so name your files in the order you want them run.
+
+### Names of manifest files:
+
+If you are a normal operator (don’t need to run before the kube apiserver), you should name your manifest files in a way that feels easy:
+
+/manifests/
+  deployment.yaml
+  roles.yaml
+
+If you’d like to ensure your manifests are applied in order to the cluster add a numeric prefix to sort in the directory:
+
+/manifests/
+  01_roles.yaml
+  02_deployment.yaml
+
+When your manifests are added to the release payload, they’ll be given a prefix that corresponds to the name of your repo/image:
+
+/release-manifests/
+  99_ingress-operator_01_roles.yaml
+  99_ingress-operator_02_deployment.yaml
+
+## How do I get added as a special run level?
```

Comment:
This belongs as a sibling of the previous subsection, since it discusses `/manifests` content.  It should be a `###` header.